carma-messenger-core/message/message/build/install/message/bin/message

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
Same as above, only the `src` folder of the workspace should be committed. Though, seeing as this one is under carma-messenger-core, maybe this is an issue with the way the repo is setup or your local workspace is setup. Not sure how this is here.